src/applications/hca/components/VeteranInformation/AuthProfileInformation.jsx

```diff
@@ -31,20 +31,23 @@ const AuthProfileInformation = ({ user }) => {
 
       <p>This is the personal information we have on file for you.</p>
 
-      <va-card data-testid="hca-profile-card" background>
+      <va-card data-testid="hca-profile-card">
+        <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
+          Personal Information
+        </h4>
         <ul className="hca-list-style-none">
-          <li>
-            <span className="vads-u-visibility--screen-reader">Full name:</span>{' '}
+          <li className="vads-u-margin-y--1">
+            <span className="vads-u-font-weight--bold">Full name:</span>{' '}
```

Comment:
Lets turn everything that is `<span className="vads-u-font-weight--bold">` to `<strong>`. That is more semantically correct.